### PR TITLE
Jupyter notebook scaling update

### PR DESCRIPTION
**Pull request by tolga-b**
Batch examples during transfer to the frontend from the backend such that we do not exceed websocket limit (I think 10MB). Tested for up to 500k examples in jupyter notebook with current changes in chrome with census notebook.

### CODE REVIEW

File: witwidget/notebook/jupyter/js/lib/wit.js [hunk 1]
```diff
@@ -143,9 +145,36 @@ var WITView = widgets.DOMWidgetView.extend({
       this.touch();
     });
     this.setupComplete = true;
+    // Notify backend that setup is complete.
+    const setupComplete = this.model.get('frontend_ready');
+    this.model.set('frontend_ready', setupComplete + 1);
```
Comment (jameswex) [inline witwidget/notebook/jupyter/js/lib/wit.js:150]: adding 1 to true?

File: witwidget/notebook/jupyter/js/lib/wit.js [hunk 2]
```diff
@@ -143,9 +145,36 @@ var WITView = widgets.DOMWidgetView.extend({
...
+  appendStagedExamples: function() {
+    if (!this.setupComplete) {
+      if (this.isViewReady()) {
+        this.setupView();
+      }
+      requestAnimationFrame(() => this.appendStagedExamples());
+      return;
+    }
+    const i = this.model.get('examples_batch_id')
+    console.log('Remaining batches: ' + i + 'loaded: ' + this.stagedExamples.length);
```
Comment (jameswex) [inline witwidget/notebook/jupyter/js/lib/wit.js:164]: remove comments when complete

File: witwidget/notebook/jupyter/js/lib/wit.js [hunk 3]
```diff
@@ -25,7 +25,7 @@ var WITView = widgets.DOMWidgetView.extend({
     this.loadAndCreateWhatIfToolElement();
 
     // Add listeners for changes from python.
-    this.model.on('change:examples', this.examplesChanged, this);
+    this.model.on('change:examples_batch', this.appendStagedExamples, this);
     this.model.on('change:config', this.configChanged, this);
     this.model.on('change:inferences', this.inferencesChanged, this);
```
Comment (jameswex) [inline witwidget/notebook/jupyter/js/lib/wit.js:30]: need to add same batching logic to inferences (indices, per-model inference results, and per-model extra outputs such as attributions). see similar logic in merged colab version of this PR (https://github.com/PAIR-code/what-if-tool/pull/120)

**Comment by tolga-b** [comment]
Closing this as #144 completes this with inferences.

**Pull request closed without merging**